dynawo/sources/ModelicaCompiler/compileCppModelicaModelInDynamicLib.cmake.in

```diff
@@ -111,6 +111,15 @@ find_package(Boost 1.64 REQUIRED
 set(SUITESPARSE_HOME "$ENV{DYNAWO_SUITESPARSE_INSTALL_DIR}")
 find_package(SuiteSparse REQUIRED)
 
+if(DEFINED ENV{DYNAWO_NICSLU_INSTALL_DIR})
```

Comment:
Do we really need the NICSLU when compiling models?

Reply:
ModelManager depends on dynawo_SolverKINSOL which depends on NICLSU